applications/endoscopy_tool_tracking/README.md

```diff
@@ -109,3 +109,7 @@ There are a two launch profiles configured for this application:
 
 1. **(debugpy) endoscopy_tool_tracking/python**: This launch profile enables debugging of Python code.
 2. **(pythoncpp) endoscopy_tool_tracking/python**: This launch profile enables debugging of Python and C++ code.
+
+## Containerize the application
```

Comment:
Neat! Could you please add another sentence or two explaining why someone might want to containerize their application? Build on one machine and distribute to another of the same architecture?

Reply:
I added a few links to reference CLI sections in the user guide instead. Please let me know if that's fine.

Reply:
Looks good, thanks!